packages/core/forms/src/composables/useAbstractFields.ts

```diff
@@ -75,6 +75,19 @@ export default function useAbstractFields(formData: {
     },
   })
 
+  // watch for programmatic changes to the model
+  watch(() => formData.model, (newVal, oldVal) => {
```

Comment:
question: since `formData.model` is an object, I imagine you should add `deep: true` to the watcher?